storage/src/test/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfigTest.java

```diff
@@ -32,21 +36,28 @@ public TestConfig(Map<?, ?> originals) {
         }
     }
 
-    @Test
-    public void testValidConfigs() {
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void testValidConfigs(boolean useDefaultRemoteLogMetadataManagerClass) {
         String rsmPrefix = "__custom.rsm.";
         String rlmmPrefix = "__custom.rlmm.";
         Map<String, Object> rsmProps = Collections.singletonMap("rsm.prop", "val");
         Map<String, Object> rlmmProps = Collections.singletonMap("rlmm.prop", "val");
+        String remoteLogMetadataManagerClass = useDefaultRemoteLogMetadataManagerClass ? DEFAULT_REMOTE_LOG_METADATA_MANAGER_CLASS_NAME : "dummy.remote.log.metadata.class";
+        System.out.println(remoteLogMetadataManagerClass);
```

Comment:
Should be removed.

Reply:
Oops, really sorry this got checked in :( Removed